packages/internal-helpers/src/remote.ts

```diff
@@ -42,7 +71,15 @@ export function matchHostname(url: URL, hostname?: string, allowWildcard?: boole
 	return false;
 }
 
-export function matchPathname(url: URL, pathname?: string, allowWildcard?: boolean) {
+/**
+ * Matches a given URL's pathname against a specified pattern, with optional support for wildcards.
+ *
+ * @param {URL} url - The URL object containing the pathname to be matched.
+ * @param {string} [pathname] - The pathname pattern to match the URL against.
+ * @param {boolean} [allowWildcard] - Determines whether wildcard matching is allowed.
```

Comment:
Same question whether there is a default here?

Reply:
Added